examples/SESAR/test/iSamplesEOI00002HBasic.json

```diff
@@ -3,20 +3,20 @@
     "@id": "https://data.isamples.org/digitalsample/igsn/EOI00002H",
     "label": "J730-GTHFS-16",
     "sampleidentifier": "igsn:EOI00002H",
-    "description": "",
+    "description": "Not Provided",
     "hasContextCategory": ["Subsurface fluid reservoir"],
     "hasMaterialCategory": ["Gaseous material"],
     "hasSpecimenCategory": ["Container with fluid"],
     "keywords": ["Individual Sample"],
     "producedBy": {
         "label": "Sampler:Fluid:GTHFS",
-        "description": "cruiseFieldPrgrm:TN300. launchPlatformName:Jason II. Sampler:Fluid:GTHFS.  HFS gastight. Red-center-9. T=250C. launch type:ROV, navigation type:USBL",
+        "description": "cruiseFieldPrgrm:TN300. launchPlatformName:Jason II. Sampler:Fluid:GTHFS. HFS gastight. Red-center-9. T=250C. launch type:ROV, navigation type:USBL",
         "hasFeatureOfInterest": "volcano",
-        "responsibility": ["Evans Leigh,,Collector","Andra Bobbitt,,Sample Owner"],
+        "responsibility": ["Evans_Leigh,,Collector","Andra Bobbitt,,Sample Owner"],
         "resultTime": "2013-09-14 01:30:00",
         "samplingSite": {
             "description": "Trevi:Jason Tmax=257.9 C. In the direct flow at this small anhydrite mound (anhydrite knocked over).",
-            "label": "",
+            "label": "Not Provided",
```

Comment:
Given that one of the sample files had this, we needed to be consistent here and use this in lieu of the empty string.